templates/repo/actions/runs_list.tmpl

```diff
@@ -7,7 +7,7 @@
 			<div class="issue-item-main gt-f1 gt-fc gt-df">
 				<div class="issue-item-top-row">
 					<a class="index gt-ml-0 gt-mr-2" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
-						{{.Title}}
+						{{- .Title -}}
```

Comment:
What's exactly the meaning of this `- <something> -`?
Does it remove trailing spaces starting from `>` until the `<`, or what?

Reply:
> What's exactly the meaning of this `- <something> -`? Does it remove trailing spaces starting from `>` until the `<`, or what?

Yes. ref https://pkg.go.dev/text/template#hdr-Text_and_spaces